src/openzeppelin/tests/mocks/reentrancy_attacker_mock.cairo

```diff
@@ -1,21 +1,23 @@
-#[abi]
-trait IAttacker {
-    fn call_sender();
+#[starknet::interface]
```

Comment:
If we are not using this Dispatcher, I think we can remove this interface, and implement the method directly as external in the mock for simplicity.